<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote AC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempUpButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempDownButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fan</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>


